### PR TITLE
✔️ Disable two flaky tests

### PR DESCRIPTION
**Pull request by danielrozenberg**
e.g.,

test-amp-ad-fake.js: https://travis-ci.org/ampproject/amphtml/jobs/509933790#L1519-L1524

test-resources.js: https://travis-ci.org/ampproject/amphtml/jobs/509628807#L1545-L1547

### CODE REVIEW

File: test/integration/test-amp-ad-fake.js [hunk 1]
```diff
@@ -32,7 +32,7 @@ describe('A4A', function() {
       `,
     extensions: ['amp-ad'],
   }, () => {
-    it('should layout amp-img, amp-pixel, amp-analytics', () => {
+    it.skip('should layout amp-img, amp-pixel, amp-analytics', () => {
```
Comment (choumx) [inline test/integration/test-amp-ad-fake.js]: `// TODO(username, #12345): Flaky on browser X under scenario Y.`

File: test/unit/test-resources.js [hunk 2]
```diff
@@ -1445,8 +1445,9 @@ describe('Resources discoverWork', () => {
   });
 
   // TODO (#16156): this test results in too many calls to getRect on Safari
-  it.configure().skipSafari()
-      .run('should update inViewport before scheduling layouts', () => {
+  // it.configure().skipSafari().run(
+  it
+      .skip('should update inViewport before scheduling layouts', () => {
```
Comment (choumx) [inline test/unit/test-resources.js]: Weird line break?

File: test/integration/test-amp-ad-fake.js (hunk 1 above)
Comment (zombifier) [inline test/integration/test-amp-ad-fake.js]: The test that's flaking is not this one, but the one at https://github.com/ampproject/amphtml/blob/master/test/integration/test-amp-inabox.js#L164.

Though I'm not sure why that flakes specifically; all the other BTF tests with more exotic setups doesn't.

File: test/unit/test-resources.js (hunk 2 above)
Comment (danielrozenberg) [reply test/unit/test-resources.js]: Wanted to minimize changing extra lines but I guess it's not a big deal. Fixing

File: test/integration/test-amp-ad-fake.js (hunk 1 above)
Comment (danielrozenberg) [reply test/integration/test-amp-ad-fake.js]: 😱 thanks for finding this! >__< embarrassing of me to disable the wrong flaky test...

File: test/integration/test-amp-inabox.js [hunk 3]
```diff
@@ -131,8 +131,8 @@ describe('inabox', function() {
         <script src="/examples/inabox-tag-integration.js"></script>
         `,
     }, env => {
-      it('should layout amp-img, amp-pixel, ' +
-          'amp-analytics', () => {
+      // TODO(zombifier, #21545): fix this flaky test
+      it.skip('should layout amp-img, amp-pixel, amp-analytics', () => {
```
Comment (zombifier) [inline test/integration/test-amp-inabox.js]: It's the test at line 164 (under 'AMPHTML ads rendered on non-AMP page BTF' that's skipping for Safari) of this file that is flaking, not this one.

Apologies for the confusion; they're all named the same even across multiple files, and only their parents are different.

Comment (danielrozenberg) [reply test/integration/test-amp-inabox.js]: Oh man, I'm acting like a drunk build cop...

Comment (danielrozenberg) [reply test/integration/test-amp-inabox.js]: OK now?

Comment (zombifier) [reply test/integration/test-amp-inabox.js]: It is now, thanks. Maybe I should rename them later on :laughing:

**Pull request merged**